quantum/quantum.c

```diff
@@ -911,40 +911,39 @@ void update_tri_layer(uint8_t layer1, uint8_t layer2, uint8_t layer3) {
   layer_state_set(update_tri_layer_state(layer_state, layer1, layer2, layer3));
 }
 
+/* Return a pseudorandom integer in the range [low, high) */
+static int randrange(int low, int high) {
+#ifdef __AVR_ATmega32U4__
+  return (TCNT0+TCNT1+TCNT3+TCNT4) % 64;
```

Comment:
`rand()` should be available on ATmega32U4 as well, so I'm not sure why this was done.

I'm looking into effective ways to seed the PRNG on these chips, so this is a WIP.